project/src/main/webapp/upload-page/upload.html

```diff
@@ -5,25 +5,65 @@
     <title>Report a Case</title>
     <link rel="stylesheet" href="upload_style.css">
     <script src="upload_script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+
   </head>
   <body>
       <div id="content"> 
-        <form action="index.html">
+        <form action="../index.html">
             <input type="submit" value="Return to Main Page" />
         </form>
-        <center>
-            <h1>Upload a report!</h1>
-            <form action="/report-handler" method="POST" enctype="multipart/form-data">
-            <!-- <form action="/report-upload" method="POST" enctype="multipart/form-data"></form> -->
-                <p>Please upload a quick description of the collision:</p>
-                <textarea name="text-input">E.g. date, time-of-day, car types of those involved, etc.</textarea>
+        <center><h1>Upload a report!</h1></center>
+        <form action="/upload" method="POST" enctype="multipart/form-data">
+        <!-- contact info, title, latitude, longitude, date+time (use html component), img url -->
+            <p>Please complete the following details of the report:</p>
+            <ul>
+                <li id = "inline-textbox">
```

Comment:
I see this id="inline-textbox" being used more than once. In CSS, IDs are used to target a single specific element. If you want to target multiple elements use a class or change directly the element type.

Also please remove the spaces before and after =. 

Apply changes to any other li element.